kxml_compiler/tests/CMakeLists.txt

```diff
@@ -1,112 +1,80 @@
-MACRO (KODE_ADD_LOCAL_XML_PARSER _sources _current_file)
-  get_filename_component( _schema ${_current_file} ABSOLUTE )
-  get_filename_component( _basename ${_schema} NAME_WE )
+MACRO(KODE_ADD_LOCAL_XML_PARSER _sources _current_file)
+	get_filename_component(_schema ${_current_file} ABSOLUTE)
+	get_filename_component(_basename ${_schema} NAME_WE)
 
-  set( _source_cpp ${CMAKE_CURRENT_BINARY_DIR}/${_basename}.cpp )
-  set( _source_h   ${CMAKE_CURRENT_BINARY_DIR}/${_basename}.h )
+	set(_source_cpp ${CMAKE_CURRENT_BINARY_DIR}/${_basename}.cpp)
+	set(_source_h ${CMAKE_CURRENT_BINARY_DIR}/${_basename}.h)
 
-  add_custom_command(
-    OUTPUT ${_source_cpp} ${_source_h}
-    COMMAND "${CMAKE_CURRENT_BINARY_DIR}/../kxml_compiler"
-    ARGS ${_schema} ${ARGN}
-    MAIN_DEPENDENCY ${_schema}
-    DEPENDS "${CMAKE_CURRENT_BINARY_DIR}/../kxml_compiler"
-  )
+	add_custom_command(
+		OUTPUT ${_source_cpp} ${_source_h}
+		COMMAND "${CMAKE_CURRENT_BINARY_DIR}/../kxml_compiler"
+		ARGS ${_schema} ${ARGN}
+		MAIN_DEPENDENCY ${_schema}
+		DEPENDS "${CMAKE_CURRENT_BINARY_DIR}/../kxml_compiler"
+	)
 
-  set( ${_sources} ${${_sources}} ${_source_cpp} )
-ENDMACRO (KODE_ADD_LOCAL_XML_PARSER)
+	set(${_sources} ${${_sources}} ${_source_cpp})
+ENDMACRO(KODE_ADD_LOCAL_XML_PARSER)
 
-set( EXECUTABLE_OUTPUT_PATH ${CMAKE_CURRENT_BINARY_DIR} )
 
-########### next target ###############
+set(EXECUTABLE_OUTPUT_PATH ${CMAKE_CURRENT_BINARY_DIR})
 
-set( testfeatures_SRCS testfeatures.cpp )
-kode_add_local_xml_parser( testfeatures_SRCS data/kde-features.rng )
+find_package(Qt5Test REQUIRED)
 
 
-#add_executable( testfeatures ${testfeatures_SRCS} )
+# testfeatures
+# FIXME BROKEN
 
-#qt5_use_modules(testfeatures Core)
-#qt5_use_modules(testfeatures Xml)
-#qt5_use_modules(testfeatures Network)
+set(testfeatures_SRCS testfeatures.cpp)
+kode_add_local_xml_parser(testfeatures_SRCS data/kde-features.rng)
+#add_executable(testfeatures ${testfeatures_SRCS})
+#target_link_libraries(testfeatures Qt5::Core Qt5::Network	Qt5::Xml)
 
-#target_link_libraries(testfeatures Qt5::Core)
-#target_link_libraries(testfeatures Qt5::Xml)
-#target_link_libraries(testfeatures Qt5::Network)
 
-#target_link_libraries( testfeatures  ${QT_QTXML_LIBRARY} )
+# testholidays
+# FIXME BROKEN
 
-########### next target ###############
+set(testholidays_SRCS testholidays.cpp)
+kode_add_local_xml_parser(testholidays_SRCS data/kde-holidays.rng)
+#add_executable(testholidays ${testholidays_SRCS})
+#target_link_libraries(testholidays	Qt5::Core	Qt5::Network Qt5::Xml)
 
-set( testholidays_SRCS testholidays.cpp )
-kode_add_local_xml_parser( testholidays_SRCS data/kde-holidays.rng )
 
-add_executable( testholidays ${testholidays_SRCS} )
-qt5_use_modules(testholidays Core)
-qt5_use_modules(testholidays Xml)
-qt5_use_modules(testholidays Network)
+# parserxsdtest
+# FIXME FAILING
```

Comment:
Use QEXPECT_FAIL in the test so that it actually "passes". Tests should never fail.